Python/Product/TestAdapter.Executor/TestAdapter.Executor.csproj

```diff
@@ -58,8 +58,12 @@
     <Reference Include="Microsoft.VisualStudio.Interop">
       <IncludeInVSIX>False</IncludeInVSIX>
     </Reference>
-    <Reference Include="Microsoft.VisualStudio.TestPlatform.ObjectModel">
-      <HintPath>$(DevEnvDir)\CommonExtensions\Microsoft\TestWindow\Microsoft.VisualStudio.TestPlatform.ObjectModel.dll</HintPath>
+    <!-- <Reference Include="Microsoft.VisualStudio.TestPlatform.ObjectModel">
+      <HintPath>$(DevEnvDir)\Extensions\TestPlatform\Microsoft.VisualStudio.TestPlatform.ObjectModel.dll</HintPath>
+      <Private>False</Private>
+    </Reference> -->
+    <Reference Include="Microsoft.TestPlatform.ObjectModel">
+      <HintPath>$(PackagesPath)\Microsoft.TestPlatform.ObjectModel.17.4.0-preview-20221003-03\lib\net462\Microsoft.VisualStudio.TestPlatform.ObjectModel.dll</HintPath>
```

Comment:
do we still need these hint paths? or was that just for testing locally with a custom testplatform lib?